src/SFA.DAS.EAS.Portal/Application/Services/Commitments/CommitmentsService.cs

```diff
@@ -0,0 +1,30 @@
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using SFA.DAS.Commitments.Api.Types.Commitment;
+using SFA.DAS.EAS.Portal.Application.Services.Commitments.Http;
+using SFA.DAS.Http;
+
+namespace SFA.DAS.EAS.Portal.Application.Services.Commitments
+{
+    public class CommitmentsService : ICommitmentsService
+    {
+        private readonly ILogger<CommitmentsService> _log;
+        private readonly RestHttpClient _restHttpClient;
+
+        public CommitmentsService(ICommitmentsApiHttpClientFactory commitmentsApiHttpClientFactory, ILogger<CommitmentsService> log)
+        {
+            _log = log;
+            _restHttpClient = new RestHttpClient(commitmentsApiHttpClientFactory.CreateHttpClient());
+        }
+
+        public Task<CommitmentView> GetProviderCommitment(long providerId, long commitmentId, 
```

Comment:
I like all params to be on separate lines if you have to split at all
`public Task<CommitmentView> GetProviderCommitment(
                                                                                                long providerId, 
                                                                                                long commitmentId, 
                                                                                                CancellationToken cancellationToken = default)`

Reply:
I tend to split lines if they go past the 120 chars visual guide, but not too worried about formatting 😄. I've tidied up the formatting, putting params onto separate lines as you like.